### ISSUE TITLE
xmax_adjust in viewClade not working

### ISSUE THREAD

**Issue by jiangweiyao** [GuangchuangYu/ggtree#187]
I think the xmax_adjust in the viewClade source code is not working.

Right now,  the code is 
"    tree_view + coord_cartesian(xlim=c(cpos$xmin, xmax), ylim=c(cpos$ymin, cpos$ymax), expand=FALSE)"

I think it should be
"    tree_view + coord_cartesian(xlim=c(cpos$xmin, xmax+xmax_adjust), ylim=c(cpos$ymin, cpos$ymax), expand=FALSE)

The code section is below

#################################

viewClade <- function(tree_view=NULL, node, xmax_adjust=0) {
    tree_view %<>% get_tree_view
    ## xd <- tree_view$data$branch.length[node]/2

    cpos <- get_clade_position(tree_view, node=node)
    xmax <- ggplot_build(tree_view)$layout$panel_ranges[[1]]$x.range[2]

    attr(tree_view, 'viewClade') <- TRUE
    attr(tree_view, 'viewClade_node') <- node

    ## tree_view+xlim(cpos$xmin, xmax + xmax_adjust) + ylim(cpos$ymin, cpos$ymax)
    tree_view + coord_cartesian(xlim=c(cpos$xmin, xmax), ylim=c(cpos$ymin, cpos$ymax), expand=FALSE)
}

**Comment by GuangchuangYu** [comment]
thanks for your report, and it has been fixed.

**Comment by matbeale** [comment]
Hi, 
I've just updated to the latest version of ggtree, and viewClade has stopped working - I think it is may be related to this issue. 

```{r} 
nwk <- system.file("extdata", "sample.nwk", package="treeio")
tree <- read.tree(nwk)
ggtree(tree) + geom_text2(aes(subset=!isTip, label=node), hjust=-.3)

```

![image](https://user-images.githubusercontent.com/16351605/43771044-e47cb566-9a35-11e8-8728-7dd874d71a8b.png)

```{r}
viewClade(ggtree(tree),17)

```
![image](https://user-images.githubusercontent.com/16351605/43771077-f91c4b44-9a35-11e8-9bce-03a2dc7983fd.png)





R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: OS X El Capitan 10.11.6

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] treeio_1.4.2         BiocInstaller_1.30.0 bindrcpp_0.2.2       rPinecone_0.1.0      randomcoloR_1.1.0    gridExtra_2.3       
 [7] reshape2_1.4.3       dplyr_0.7.6          plyr_1.8.4           phytools_0.6-44      maps_3.3.0           ggtree_1.12.6       
[13] ape_5.1              ggplot2_3.0.0

**Comment by GuangchuangYu** [comment]
@matbeale fixed in ggtree `v>=1.12.7`.
